podcastparser.py

```diff
@@ -686,6 +686,8 @@ def set_base(self, base):
         self.base = base
 
     def set_podcast_attr(self, key, value):
+        if key == 'type' and value not in ['episodic', 'serial']:
```

Comment:
This can be a tuple: `... not in ('episodic', 'serial')`

It shouldn't matter performance-wise in recent Python 3 versions (see [this Stackoverflow question](https://stackoverflow.com/questions/25368337/tuple-or-list-when-using-in-in-an-if-clause)) but IMHO is "clearer".